### ISSUE TITLE
[Reputation] Sons of Hodir

### ISSUE THREAD

**Issue by Enteroox** [gamefreedomgit/Frostmourne#1010]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (https://www.wowhead.com/wotlk)

**Description:**
Upon completing the quest [A Spark of Hope](https://www.wowhead.com/wotlk/quest=12956/a-spark-of-hope) with [Thorim](https://www.wowhead.com/wotlk/npc=29445/thorim), you move up from "Neutral" to "Revered" with [The Sons of Hodir](https://www.wowhead.com/wotlk/faction=1119/the-sons-of-hodir). And once you complete [Mending Fences](https://www.wowhead.com/wotlk/quest=12915/mending-fences) you immediately progress to "Exalted" with Sons of Hodir.
![image](https://github.com/gamefreedomgit/Frostmourne/assets/96174069/bca73e61-bf91-4bf1-ad2c-85758397a2ca)
**How to reproduce:**
Complete the quest chain that supposedly brings you to a Neutral state with Sons of Hodir and upon finishing the last two quests, you will achieve an Exalted reputation with Sons of Hodir.

**How it should work:**
Instead of earning Revered and Exalted, you're supposed to earn Neutral.

**Database links:**
[Thorim](https://www.wowhead.com/wotlk/npc=29445/thorim)
[The Sons of Hodir](https://www.wowhead.com/wotlk/faction=1119/the-sons-of-hodir)
[A Spark of Hope](https://www.wowhead.com/wotlk/quest=12956/a-spark-of-hope)
[Mending Fences](https://www.wowhead.com/wotlk/quest=12915/mending-fences)

**Comment by Basedmikeftw** [comment]
There are few thingst hat could cause this and are not mentioned.

Could be ancestral wisdom doing this, do you have any 80 lvl characters?
Also could be bonus weekend, I know this was posted on wednesday but there is no timestamp on the cropped image, could be taken anytime.
And lastly, if I understand correctly, you get 100% rep bonus on reps that you have exalted with on other characters.

On my second char, I waited for bonus weekend just to get that massive boost in rep (did not know about the 100% at that time).

On retail, doing Netherwing is pretty much most efficient (and recommended) when there is TBC timewalking going on, since you get 50% bonus on TBC reputations, this adds 21k rep to the massive 42k you would get that would normally take you to neutral level with them, effectively putting yourself at revered with just the intro questline, all that with just some clever use of game mechanics.

**Comment by Enteroox** [comment]
Hey Mike!

To answer your suspicions:
The bug report was created about 20 minutes after I took the screenshot. Therefore, it is from the 19th of June.
I have another character, however, I do not have any reputation achieved for him. I reached level 80 with it and decided to play a different character.
I am not aware of this Ancestral Wisdom, to be honest, so I am not sure how it may be affecting me, and why it may cause me to skip through all the reputation ranks.

Thank you for the feedback nonetheless.
Have a good day.
